src/Profile/ProfileScreen.js

```diff
@@ -170,25 +172,25 @@ export default function ProfileScreen({ navigation }) {
           Alert.alert(err.error, err.message);
         }
       });
+      const updatedUser = {
+        ...currentUser, email, address, phoneNumber: phoneNum,
+      };
+      serviceUpdateUser(updatedUser);
     } else {
       Alert.alert('Please fill out all fields.');
     }
-  }
+  };
 
-  function onCancel() {
-    console.log('cancel');
+  const onCancel = () => {
```

Comment:
this function doesn't actually reset the textfields back to their original state. when a user clicks cancel, the textfields should switch back so that a user doesn't think their info has been updated accidentally. this can be done by setting the email, address, and phoneNum variables with the currentUser data object (follow the examples I have already implemented)